app/upgrades.go

```diff
@@ -3,47 +3,29 @@ package app
 import (
 	"fmt"
 
+	"github.com/CosmWasm/wasmd/x/wasm"
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
 	store "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
-	"github.com/cosmos/cosmos-sdk/x/authz"
-	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
 // next upgrade name
-const upgradeName = "v2"
+const upgradeName = "v3"
 
 // RegisterUpgradeHandlers returns upgrade handlers
 func (app *App) RegisterUpgradeHandlers(cfg module.Configurator) {
 	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
-		const (
-			HumanCoinUnit = "stars"
-			BaseCoinUnit  = "ustars"
-			StarsExponent = 6
-		)
-		denomMetadata := banktypes.Metadata{
-			Description: "The native token of Stargaze",
-			DenomUnits: []*banktypes.DenomUnit{
-				{
-					Denom:    BaseCoinUnit,
-					Exponent: 0,
-					Aliases:  []string{"microstars"},
-				},
-				{
-					Denom:    HumanCoinUnit,
-					Exponent: StarsExponent,
-					Aliases:  nil,
-				},
-			},
-			Base:    BaseCoinUnit,
-			Display: HumanCoinUnit,
-			Name:    "Stargaze STARS",
-			Symbol:  "STARS",
+		// wasm.InitGenesis(ctx, &app.WasmKeeper, )
```

Comment:
Is this meant to be commented out?

Reply:
Its using the default genesis and just overriding the params will remove it